Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.9-bullseye
+
+ENV PIP_NO_CACHE_DIR=1
+RUN apt-get update && apt-get -y upgrade && \
+    apt-get -y install gdal-bin python3-gdal && \
+    apt-get -y autoremove && \
+    apt-get clean
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip install --upgrade pip
+RUN pip install -r requirements.docker.txt
+RUN pip install -e .
+
+COPY /src/config.txt src/config.txt
+
+HEALTHCHECK CMD curl -f http://localhost:8000 || exit 1
```

Comment:
When I run `curl -f http://localhost:8000` I get a 404 Response.

Sould we replace it by `curl -f http://localhost:8000/latest/docs` and do the same in Docker/Dockerfile ?
Or the unhealthy status is linked to the setup error I have reported above ?

Reply:
oh that's right should be /latest/docs . Thanks